lib/src/v3/models/local_user/local_user.dart

```diff
@@ -17,13 +17,15 @@ class LocalUser with _$LocalUser {
     bool? blurNsfw, // v0.18.3
     bool? autoExpand, // v0.18.3
     required String theme, // v0.18.0
-    required SortType defaultSortType, // v0.18.0
+    @deprecated SortType? defaultSortType, // v0.18.0 [deprecated in v0.20.0]
+    required SortType defaultCommentSortType, // v0.20.0
```

Comment:
For the newly added values, we won't be able to have the `required` field because it has to be compatible with the previous version (whenever possible)!

```suggestion
    SortType? defaultCommentSortType, // v0.20.0
```

Reply:
Oh got it, that makes sense.